src/Container/TemplatedErrorHandlerFactory.php

```diff
@@ -38,6 +38,15 @@
  * </code>
  *
  * If any of the keys are missing, default values will be used.
+ *
+ * @deprecated Since 2.0, to remove in 3.0. Please migrate your code to use
```

Comment:
Hmm... This is tricky. As noted in another reply, we still need a final handler of some sort, and this is one way to provide it; the problem is that the implementation is largely irrelevant due to the fact that the `$e`/`$error` argument to it will never be provided, and the `$response` will almost always be the original response or an empty response, due to the way the delegators are propagated. In other words, this factory, and the implementation it provides, is largely useless.

I'm just not sure how to message its deprecation.

Let me think on that some more before making a decision.